source/parameters/magneticmultipole.md

```diff
@@ -0,0 +1,89 @@
+%---------------------------------------------------------------------------------------------------
+(s:mag.mult.params)=
+## MagneticMultipoleP: Magnetic Multipole Parameters
+
+The `MagneticMultipoleP` parameter group describes magnetic multipoles associated with the lattice
+element. For a multipole of order `N`, the magnetic field {math}`(B_x, B_y)`
+at a point {math}`\bf r` in terms of the points polar coordinates {math}`(r, \theta)` is
+```{math}
+:label: bbmult
+
+B_x({\bf r}) + i \, B_y({\bf r}) = 
+\frac{1}{N!}(B_n + i \, B_s) \, e^{-i(N+1)T} \, e^{iN\theta} \, r^N
+```
+where {math}`T` is the "tilt" of the `N`{sup}`th` multipole and {Math}`B_n` and {math}`B_s` are
+the normal and skew components of the field. `N` is defined such that 
+{math}`N = 0` describes a dipolar multipole, {math}`N = 1` describes a quadrupolar multipole, etc.
+
+The three parameters, {math}`T`, {Math}`B_n` and {math}`B_s` are not independent and only two
+are needed to specify a multipole. 
+However, it is sometimes convenient to use all three. 
+For example, {Math}`B_n` and {math}`B_s` with {math}`N=0` can be used to describe 
+an element with independent horizontal and vertical steerings while {math}`T` can be used
+to represent rotational errors.
+
+The components of `MagneticMultipoleP` for specifying a multipolar field of order `N` is:
+```{code} yaml
+tiltN     - Tilt
+BnN       - Normal component 
+BsN       - Skew component
+```
+
+Alternatively, the field components can be specified using normalized values
+```{code} yaml
+KnN       - Normalized normal component 
+KsN       - Normalized skew component
+```
+where the conversion between field and normalized components is:
+```{math}
+  (K_n, K_s) = \frac{q}{P_0} \, (B_n, B_s)
+```
+with {math}`q` being the charge of the reference particle and {math}`P_0` being the 
+reference momentum.
+
+Furthermore, the field and normalized values can be given in terms of the integrated strength.
+Integrated values are specified with the letter 
+`L` appended at the end of the name. Example:
+```{code} yaml
+MagneticMultipoleP:
+  tilt7: 0.7        - Tilt of 7th order multiple
+  Bn3: 27.3         - Normal multipole component of order 3.
+  Bn3L: 3.47e1      - Length integrated normal multipole component of order 3.
+```
+The length integrated values are related to the non-integrated values via
+```{math}
+  (B_n, B_s, K_n, K_s) = L \, (B_n, B_s, K_n, K_s)
+```
+where `L` is the length of the element.
```

Comment:
I wonder if there is a way to make the notation in this formula clearer?

Here I see $(B_n,B_s,K_n,K_s) = L(B_n,B_s,K_n,K_s)$. For one single component this reads, e.g., $B_n = L B_n$, which I think may be not too clear, given that the quantity $B_n$ appears on both sides of the equation.

Also, if the sentence "where `L` is the length of the element" refers to the equation, maybe we could write "L" in math style rather than code style.

Reply:
Maybe the length-integrated values could be explicitly defined by `BnNL` = _L_*`BnN`, or something similar.

Reply:
Yes this was a typo with the `L` left off. This has been fixed